dotnet/src/Skills/Skills.Web/Bing/BingConnector.cs

```diff
@@ -74,23 +75,23 @@ public void Dispose()
 
     [SuppressMessage("Performance", "CA1812:Internal class that is apparently never instantiated",
         Justification = "Class is instantiated through deserialization.")]
```

Comment:
Now that these classes are used in the source generator (and thus instantiated by C# code), I assume we can get rid of these SuppressMessages?